feature-group-definitions/blocking-render.yml

```
@@ -0,0 +1,11 @@
+name: 'blocking="render"'
+details: 'The `blocking="render"` attribute for `<link>`, `<script>`, and `<style>` elements blocks rendering until the external script or stylesheet has been loaded.'
```

This'll fix the test

```suggestion
description: 'The `blocking="render"` attribute for `<link>`, `<script>`, and `<style>` elements blocks rendering until the external script or stylesheet has been loaded.'
```